apps/astro/astro.config.mjs

```diff
@@ -15,10 +15,9 @@ export default defineConfig({
       dataset,
       useCdn: true,
       apiVersion,
-      studioUrl: ({_dataset}) =>
-        _dataset === workspaces['cross-dataset-references'].dataset
-          ? `${baseUrl}/${workspaces['cross-dataset-references'].workspace}`
-          : studioUrl,
+      stega: {
+        studioUrl,
+      },
```

Comment:
I had to change this because `@sanity/astro` doesn't support a function here (I've shared more info about this in Slack). Also, `studioUrl` is now **inside** the `stega` config.

Reply:
Could you link to the slack thread? 🙌

Reply:
We don't have support for cross dataset references yet in Presentation anyway, so we can revisit this later once we land that :)

Reply:
> Could you link to the slack thread? 🙌

Here you go! https://sanity-io.slack.com/archives/C02CRE5GS2K/p1715715049501499